LICENSES/MIT.txt

```diff
@@ -0,0 +1,18 @@
+MIT License
+
+Copyright (c) <year> <copyright holders>
```

Comment:
does this need the original copyright year/holders from the old LICENSE file

Reply:
REUSE recommends not to:
https://reuse.software/faq/#license-templates

But the project needs to make sure to keep all relevant copyright holders in the copyright tags per file.